I don't think the failures are related to this change

HibernateOrmCodestartTest failure example:
```
2023-01-19T16:25:36.1778852Z [ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 7.228 s <<< FAILURE! - in io.quarkus.devtools.codestarts.quarkus.HibernateOrmCodestartTest
2023-01-19T16:25:36.1785810Z [ERROR] io.quarkus.devtools.codestarts.quarkus.HibernateOrmCodestartTest.testContent  Time elapsed: 0.062 s  <<< FAILURE!
2023-01-19T16:25:36.1786507Z java.lang.AssertionError: 
2023-01-19T16:25:36.1787353Z [Snapshot is not matching (use -Dsnap to update it automatically): HibernateOrmCodestartTest/testContent/src_main_kotlin_ilove_quark_us_MyKotlinEntity.kt] 
2023-01-19T16:25:36.1787872Z Path:
2023-01-19T16:25:36.1788583Z   target/quarkus-codestart-test/quarkus-jdbc-h2-hibernate-orm-4fc4c3f6-cb4f-402f-a0c4-771108090ed4/mocked-data/kotlin/src/main/kotlin/ilove/quark/us/MyKotlinEntity.kt
2023-01-19T16:25:36.1789106Z and path:
2023-01-19T16:25:36.1789952Z   /home/runner/work/quarkus/quarkus/integration-tests/devtools/target/test-classes/__snapshots__/HibernateOrmCodestartTest/testContent/src_main_kotlin_ilove_quark_us_MyKotlinEntity.kt
2023-01-19T16:25:36.1790458Z do not have same content:
2023-01-19T16:25:36.1790674Z 
2023-01-19T16:25:36.1790837Z Changed content at line 12:
2023-01-19T16:25:36.1791144Z expecting:
2023-01-19T16:25:36.1791409Z   [" * ```kotlin"]
2023-01-19T16:25:36.1791653Z but was:
2023-01-19T16:25:36.1791895Z   [" * {@code"]
2023-01-19T16:25:36.1792081Z 
2023-01-19T16:25:36.1792242Z Changed content at line 23:
2023-01-19T16:25:36.1792529Z expecting:
2023-01-19T16:25:36.1792774Z   [" * ```"]
2023-01-19T16:25:36.1793030Z but was:
2023-01-19T16:25:36.1793265Z   [" * }"]
```

--- a/core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java
@@ -213,7 +213,7 @@ public class NativeConfig {
      * The docker image to use to do the image build. It can be one of `graalvm`, `mandrel`, or the full image path, e.g.
      * {@code quay.io/quarkus/ubi-quarkus-mandrel-builder-image:22.3-java17}.
      */
-    @ConfigItem(defaultValue = "${platform.quarkus.native.builder-image}")
+    @ConfigItem(defaultValue = "${platform.quarkus.native.builder-image}", defaultValueDocumentation = "mandrel")
     public String builderImage;
 
     public String getEffectiveBuilderImage() {

